lib/transport/selenium-webdriver/browserstack.js

```diff
@@ -42,18 +59,18 @@ class Browserstack extends Selenium {
     this.settings.webdriver.port = Number(this.settings.webdriver.port);
 
     if (!this.accessKey && process.env.BROWSERSTACK_KEY) {
-      this.settings.desiredCapabilities['browserstack.key'] = process.env.BROWSERSTACK_KEY;
+      this.settings.desiredCapabilities['browserstack.key'] = browserstackKey;
     }
 
     if (!this.username && process.env.BROWSERSTACK_USER) {
-      this.settings.desiredCapabilities['browserstack.user'] = process.env.BROWSERSTACK_USER;
+      this.settings.desiredCapabilities['browserstack.user'] = browserstackUsername;
```

Comment:
`browserstackUsername` is never assgined a value

Reply:
fixed